ppl/src/test/java/org/opensearch/sql/ppl/parser/AstBuilderTest.java

```diff
@@ -282,6 +282,27 @@ public void testStatsCommandWithByClause() {
         ));
   }
 
+  @Test
+  public void testStatsCommandWithByClauseInBackticks() {
+    assertEqual("source=t | stats count(a) by `b` DEDUP_SPLITVALUES=false",
```

Comment:
If the query is `"source = t | stats count(a) by `t.b`"` it would treat t.b as column name.